daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/TryOps.scala

```diff
@@ -17,6 +17,7 @@ private[daml] object TryOps {
       b <- tb
     } yield f(a, b)
 
+  @deprecated("use scala.util.Using", since = "1.16.0")
```

Comment:
I think this is used only internally, any reason to leave it around?

Reply:
I just prefer to deprecate anything which is public.